src/sdk/namespace_fs.js

```diff
@@ -333,6 +333,11 @@ class NamespaceFS {
             const file_path = this._get_file_path(params);
             const stat = await fs.promises.stat(file_path);
             console.log(file_path, stat);
+            if (stat.isDirectory()) {
```

Comment:
@eshelmarc A comment on this condition block will com in handy for future readers. Can you please add a short explanation? Thanks